test/unit/lib/entities/context/DutchQuoteContext.test.ts

```diff
@@ -335,6 +335,82 @@ describe('DutchQuoteContext', () => {
       expect((quote?.toJSON() as DutchQuoteDataJSON).orderInfo.exclusiveFiller).toEqual(filler);
     });
 
+    it('uses synthetic if worse with forceOpenOrders=true and switch=false', async () => {
```

Comment:
nit: wdym by `switch=false`?

Reply:
There used to be a service that URA would call that would say whether synthetic orders were enabled or not: `SyntheticStatusProvider` - it's now a noop in the code with `DisabledSyntheticStatusProvider`. This test (and others like it use `makeProviders(false)` to simulate a false response from that API.